util/displayErrors.js

```diff
@@ -1,19 +1,23 @@
 module.exports = function displayErrors(err, type){
-	const response = JSON.parse(err.requestResult.responseRaw);
-	const errors = response.errors;
+	if (err.requestResult) {
```

Comment:
Minor detail but when I was testing I was experiencing errors that were being hidden because they were normal JS errors and not errors with the FaunaDB response itself. This change just checks for the `requestResult` property on the error before trying to parse it.